website/docs/development/compiling.md

```
@@ -46,9 +52,15 @@ For **release builds** (no logging, optimized), use `release.conf` instead of `d
 
+   **For the native simulator** (`native_sim/native/64`), you also need to add under **Extra CMake arguments**:
+   ```
+   -DSB_CONF_FILE=sysbuild_no_mcuboot_no_xip.conf
+   ```
+
 4. Click **Generate and Build**.
 
-## Option B: Build from Command Line
+  </TabItem>
+  <TabItem value="cli" label="Command Line">
 
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)